### PR TITLE
feat: perform console oauth in the frontend only

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: console-frontend/src/pages/devices/index.rs [hunk 1]
```diff
@@ -273,7 +272,7 @@ impl Index {
     }
 
     fn load_apps(&self, ctx: &Context<Self>) -> Result<RequestHandle, anyhow::Error> {
```
Comment (jbtrystram) [inline console-frontend/src/pages/devices/index.rs:274]: If reqwest can be built to WASM target, we can use drogue-client here as well

Comment (ctron) [reply console-frontend/src/pages/devices/index.rs:274]: True. But I would like to do that as a follow up task. This PR is already rather big, bigger than anticipated.

**Pull request merged**